### DISCUSSION TITLE
Multiple developers' mode.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/Microsoft-Fabric-workload-development-sample#118] [Q&A] [answered]
What would be the ideal way to use DevGateway, when multiple developers want to test their backend in their local?

**Comment by michal-mano** [comment, marked as answer]
Hello,
We plan to support this scenario in mid-November (more details will be announced in the coming weeks).
Currently, each user must have their own capacity to use with the DevGateway, or multiple developers must use the same user.
